.github/pr-commenter.yml

```diff
@@ -0,0 +1,67 @@
+      files:
+        - '*'
+        - '**/*'
+      body: |
+        - **General steps**
+            - :trophy: Does this PR need to receive a label with a modifier (`reputation/contributed_code/{minor,major}`)? (A medium reputation amount is awarded by default, see [docs](https://github.com/exercism/docs/blob/main/product/reputation.md#4-creating-pull-requests))
+
+    - id: any-exercise-changed
+      files:
+        - 'exercises/**/*'
+      body: |
+        - **Any exercise changed**
+            - :bust_in_silhouette: Does the author of the PR need to be added as an author or contributor in `<exercise>/.meta/config.json` (see [docs](https://github.com/exercism/docs/blob/main/building/tracks/practice-exercises.md#file-metaconfigjson))?
+            - :microscope: Do the [analyzer](https://github.com/exercism/elixir-analyzer/tree/main/test/elixir_analyzer/test_suite) and the [analyzer comments](https://github.com/exercism/website-copy/tree/main/analyzer-comments/elixir) exist for this exercise? Do they need to be changed?
+            - :scroll: Does the design file (`<exercise>/.meta/design.md`) need to be updated to document new implementation decisions?
+
+    - id: practice-exercise-changed
+      files:
+        - 'exercises/practice/**/*'
+      body: |
+        - **Practice exercise changed**
+            - :evergreen_tree: Do `prerequisites`, `practices`, and `difficulty` in `config.json` need to be updated?
+            - :teacher: Are the changes in accordance with the community [problem specifiations](https://github.com/exercism/problem-specifications/tree/main/exercises)?
+
+    - id: practice-exercise-tests-changed
+      files:
+        - 'exercises/practice/**/test/**/*'
+      body: |
+        - **Practice exercise tests changed**
+            - :white_circle: Are all tests but the first one _skipped_?
+            - :scroll: Does `<exercise>/.meta/tests.toml` need updating?
+
+    - id: concept-exercise-changed
+      files:
+        - 'exercises/concept/**/*'
+      body: |
+        - **Concept exercise changed**
+            - :evergreen_tree: Do `prerequisites` and `practices` in `config.json` need to be updated?
+            - :book: Does the concept introduction provide all necessary information to solve this exercise?
+
+    - id: concept-exercise-test-changed
+      files:
+        - 'exercises/concept/**/test/**/*'
+      body: |
+        - **Concept exercise tests changed**
+            - :white_circle: Are all tests _un-skipped_?
+
+    - id: concept-changed
+      files:
+        - 'concepts/**/*'
+      body: |
+        - **Concept changed**
+            - :bust_in_silhouette: Does the author of the PR need to be added as an author or contributor in `<concept>/.meta/config.json` (see [docs](https://github.com/exercism/docs/blob/main/building/tracks/concepts.md#file-metaconfigjson))?
+            - :book: Does the concept introduction provide all necessary information to solve this exercise?
```

Comment:
I'm presuming based on this that you use the concept introductions as the exercise introductions? When we generalise this to other tracks, we'll need to remember that some tracks don't do this, and that this should instead be in the concept-exercise block.

Reply:
Yes, we use word-for-word exactly the same concept and exercise introduction. We'll make it explicit by using the template once configlet supports it.